src/scancode/cli.py

```diff
@@ -279,6 +282,9 @@ def scancode(ctx,
 
     scans_cache_class = get_scans_cache_class()
 
+    # Convert ignore (user defined ignored files list) to a dict
+    user_ignore = dict((patt, 'User ignore: Supplied by --ignore') for patt in ignore)
```

Comment:
What is `'User ignore: Supplied by --ignore' about?
Also using a dict comprehension `{x: y for x, y in foo}` is more explicit